pkg/connections/stomp/connection_test.go

```diff
@@ -173,26 +190,26 @@ func BenchmarkPublishAndSubscribe(b *testing.B) {
 	defer c.Close()
 
 	// Subscribe to the "destination name" destination.
-	c.Subscribe("destination-name", callbackFactory(messageRecieved))
-	defer c.Unsubscribe("destination-name")
-
-	// Set a hello world message.
-	m := client.Message{
-		Data: client.MessageData{
-			"value": 42.0,
-		},
-	}
+	c.Subscribe(destination, callbackFactory(messageRecieved))
+	defer c.Unsubscribe(destination)
 
 	// Publish b.N messages.
 	for n := 0; n < b.N; n++ {
-		c.Publish(m, "destination-name")
+		// Send a message with counter as value.
+		m = client.Message{
+			Data: client.MessageData{
+				"value": n,
```

Comment:
For benchmarking we should probably use messages with a more realistic size, say 1 KiB.

Reply:
Sending 1kb payload, also updated the Screen dump in the description for this PR.